<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Resolution</class>
 <widget class="QMainWindow" name="Resolution">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>650</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>650</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>650</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Resolution</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normalon>../../EightDigitsPuzzle/ui/img/icon.png</normalon>
   </iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>1000</width>
     <height>650</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>1000</width>
     <height>650</height>
    </size>
   </property>
   <widget class="QLabel" name="Label_Copyright">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>610</y>
      <width>951</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Copyright Â© 2022 LPH. All Rights Reserved</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="Label_Title">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>0</y>
      <width>951</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lucida Handwriting</family>
      <pointsize>24</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 24pt &quot;Lucida Handwriting&quot;;</string>
    </property>
    <property name="text">
     <string>Clause Resolution</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="Label_Result">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>550</y>
      <width>951</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Rockwell</family>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>211</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Input condition below:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>80</y>
      <width>211</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Steps will show below:</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="Text_Condition">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>261</width>
      <height>431</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Adobe Devanagari</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="acceptRichText">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QTableWidget" name="Table">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>110</y>
      <width>670</width>
      <height>381</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>ID</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Source</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Expression</string>
     </property>
    </column>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>500</y>
      <width>671</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>84</number>
     </property>
     <item>
      <widget class="QPushButton" name="Button_Start">
       <property name="text">
        <string>Start</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Button_ShowPath">
       <property name="text">
        <string>Show path</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Button_Clear">
       <property name="text">
        <string>Clear</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <zorder>Button_Clear</zorder>
   <zorder>Button_Start</zorder>
   <zorder>Label_Title</zorder>
   <zorder>Label_Copyright</zorder>
   <zorder>Label_Result</zorder>
   <zorder>Button_ShowPath</zorder>
   <zorder>label</zorder>
   <zorder>label_2</zorder>
   <zorder>Text_Condition</zorder>
   <zorder>Table</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
